Update ppb defund

TessMyers opened the pull request:
The Portland City Budget was amended but not approved in a Council meeting on July 11th. This letter responds to these changes directly, following guidance from Care Not Cops PDX among other sources (formal approval from CNC is currently being sought). 

Thank you for reviewing!
Sidebar, is there any way to view the final email format before re-deploying? I have both newline and whitespace formatting in other email templates, and am not sure which I should use.

Commit:
Update PPB defund letter to respond to budget amendments

Commit:
remove newlines, add spacing

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,48 @@ recipients:
+
+We need to know that we are being taken seriously and that we mean for the PPB to be dismantled in support of building Black futures and community care. The PPB does not protect us equally and we can do better.
+
+Specifically, in accordance with the demands of Care Not Cops and the design to defund and reallocate, I ask for:
```
```suggestion
Specifically, in accordance with the design to defund and reallocate, I ask for:
```

swallan replied:
refrain from endorsing outside organizations

TessMyers replied:
My understanding is that letters should refrain from endorsing outside orgs without permission of said org. Is this a general rule (which I see the logic of), or would it be ok to keep this given specific written permission from Care Not Cops?

swallan replied:
It is a general rule

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,48 @@ recipients:
+  3.a) Redirection of these funds towards anti-violence programs unaffiliated with law enforcement, reparations for Black Portlanders targeted by gang policing, and other investment in Black people to be determined by impacted communities and grassroots groups such as Don’t Shoot Portland & Freedom to Thrive.
+  3.b) Eliminate these positions entirely by reducing the total number of officers, not just by folding them back into general sworn officer positions.
+
+4) Immediately stop policing protests - officers are bringing the riot, not the demonstrators. Even if they stop using tear gas, glass beads, plastic and rubber bullets, and other 'less lethal weapons' they will continue to behave how they know and shift to other ways to enact violence.
```
This is nice, but will have to be amended when pr
```suggestion

```

A great point for right now, but would have to be updated later, so I suggest removing until we have time senstive letters.

TessMyers replied:
Great point, thank you!

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,48 @@ recipients:
+
+6) Divest City of Portland Cannabis Tax funds from the PPB budget AND invest them instead into reparations for those harmed by the War on Drugs, which disproportionately targeted Black communities for activities that are now legal.
+
+7) Immediately stop expansion of the Portland Police Bureau budget by ending all budget increases. The City Council must commit to never vote for PPB Budget increases again.
```
move this point closer to top

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,48 @@ recipients:
+
+10) Immediately eliminate police presence in historically Black neighborhoods, communities of color, and areas with high houseless populations in order to reduce the fear, ongoing trauma, violence, and oppressive conditions which impede people’s daily survival.
+
+Finally, I need to see that there will be allocations to the bereaved families of the citizens of Portland killed by members of the PPB. I urge that the testimonies of those who have come forward be reviewed, as the names of the officers at fault and the victims that have fallen at their hands have been adequately named.
```
not completely on message

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,48 @@ recipients:
+
+The PPB has been caught in collusion with prominent white nationalist groups, shootings of Black people and people perceived to be in mental health crisis, and this violence and terrorism against our citizens needs to stop NOW.
+
+I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence ...
```
```suggestion
I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence to ensure that we nobody needs to use the Black Lives Matter hashtag anymore because it will be evident in our practices.
```

swallan commented:
> Sidebar, is there any way to view the final email format before re-deploying? I have both newline and whitespace formatting in other email templates, and am not sure which I should use.

@TessMyers: I prefer whitespace, both work

Commit:
Remove CNC reference, reorganize points, remove time-sensitive demands

bingbongle commented:
@TessMyers Please address in the next day or so, or this PR will be closed automatically.

TessMyers commented:
@bingbongle I addressed all feedback 2 days ago and was waiting for an approval. Am I missing something? I use Bitbucket for my day-to-day job rather than the github PR interface, so please let me know if I've missed a step. 

In any case, the Portland council is voting on the police budget today, so I may need to make changes to this letter afterwards.

bingbongle commented:
This formatting for the body text won't work. You'll need to either use a pipe character after body and indent everything below it, or you will have to manually add newline characters. Take a look at _emails/README.md and you'll see how to format there.

swallan commented on _emails/us/oregon/portland_defund_ppb.md:
```diff
@@ -17,34 +17,44 @@ recipients:
-  vote if you aren't willing to make a public and vocal stand to help defund the PPB
-  and increase their accountability and transparency.\n\nThank you for your time,\n[YOUR
-  NAME]\n[YOUR ADDRESS]\n[YOUR EMAIL]\n[YOUR PHONE NUMBER]"
+body: "Hello,
+
+My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
```
```suggestion
body:  |-
Hello,

My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
```
